pkg/cmd/server/origin/controller/config.go

```diff
@@ -14,10 +14,28 @@ import (
 
 	configapi "github.com/openshift/origin/pkg/cmd/server/api"
 	"github.com/openshift/origin/pkg/cmd/server/crypto"
-	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
 	"github.com/openshift/origin/pkg/cmd/util/variable"
 )
 
+func envVars(host string, caData []byte, insecure bool, bearerTokenFile string) []kapi.EnvVar {
```

Comment:
Good.  I hate having "general helpers" that are only used in one spot.  It seems like a way to just push debt to others.